contracts/FiatToken.cdc

```diff
@@ -1,1287 +1,268 @@
 
     // Minting events
-    pub event MinterCreated(resourceId: UInt64)
-    pub event MinterControllerCreated(resourceId: UInt64)
-    pub event Mint(minter: UInt64, amount: UFix64)
-    pub event Burn(minter: UInt64, amount: UFix64)
-    pub event MinterConfigured(controller: UInt64, minter: UInt64, allowance: UFix64)
-    pub event MinterRemoved(controller: UInt64, minter: UInt64)
-    pub event ControllerConfigured(controller: UInt64, minter: UInt64)
-    pub event ControllerRemoved(controller: UInt64)
-
-
-    // ------- FungibleToken Events -------
-
-    pub event TokensInitialized(initialSupply: UFix64)
-    pub event TokensWithdrawn(amount: UFix64, from: Address?)
-    pub event TokensDeposited(amount: UFix64, to: Address?)
-
+    access(all) event MinterCreated(resourceId: UInt64)
+    access(all) event Mint(minter: UInt64, amount: UFix64)
+    access(all) event Burn(minter: UInt64, amount: UFix64)
 
     // ------- FiatToken Paths -------
 
-    pub let VaultStoragePath: StoragePath
-    pub let VaultBalancePubPath: PublicPath
-    pub let VaultUUIDPubPath: PublicPath
-    pub let VaultReceiverPubPath: PublicPath
-
-    pub let BlocklistExecutorStoragePath: StoragePath
-
-    pub let BlocklisterStoragePath: StoragePath
-    pub let BlocklisterCapReceiverPubPath: PublicPath
-    pub let BlocklisterUUIDPubPath: PublicPath
-    pub let BlocklisterPubSigner: PublicPath
-
-    pub let PauseExecutorStoragePath: StoragePath
-
-    pub let PauserStoragePath: StoragePath
-    pub let PauserCapReceiverPubPath: PublicPath
-    pub let PauserUUIDPubPath: PublicPath
-    pub let PauserPubSigner: PublicPath
-
-    pub let AdminExecutorStoragePath: StoragePath
-
-    pub let AdminStoragePath: StoragePath
-    pub let AdminCapReceiverPubPath: PublicPath
-    pub let AdminUUIDPubPath: PublicPath
-    pub let AdminPubSigner: PublicPath
-
-    pub let OwnerExecutorStoragePath: StoragePath
-
-    pub let OwnerStoragePath: StoragePath
-    pub let OwnerCapReceiverPubPath: PublicPath
-    pub let OwnerUUIDPubPath: PublicPath
-    pub let OwnerPubSigner: PublicPath
-
-    pub let MasterMinterExecutorStoragePath: StoragePath
-
-    pub let MasterMinterStoragePath: StoragePath
-    pub let MasterMinterCapReceiverPubPath: PublicPath
-    pub let MasterMinterUUIDPubPath: PublicPath
-    pub let MasterMinterPubSigner: PublicPath
-
-    pub let MinterControllerStoragePath: StoragePath
-    pub let MinterControllerUUIDPubPath: PublicPath
-    pub let MinterControllerPubSigner: PublicPath
-
-    pub let MinterStoragePath: StoragePath
-    pub let MinterUUIDPubPath: PublicPath
-
+    access(all) let VaultStoragePath: StoragePath
+    access(all) let VaultBalancePubPath: PublicPath
+    access(all) let VaultReceiverPubPath: PublicPath
+    access(all) let MinterStoragePath: StoragePath
 
     // ------- FiatToken States / Variables -------
 
-    pub let name: String
-    pub var version: String
-    // Set to true if the contract is paused
-    pub var paused: Bool
```

Comment:
The `FiatToken` contract is just being integrated into the bridge so that the bridge is the only controller of the contract, so all additional functionality besides minting and burning is being removed since the bridge doesn't support any functionality besides minting and burning. I believe we already confirmed this with the Circle team